### PR TITLE
Log subsystem initialize times in feedback, give more precise displays

### PR DESCRIPTION
**Pull request by Mothblocks**
## About The Pull Request
Logs subsystem initialize times in feedback in milliseconds.

Makes chat messages about SS inits precise to 2 decimal points.

## Changelog

:cl:
qol: Subsystem initialization times now show to 2 decimal points.
/:cl:

### CODE REVIEW

File: code/controllers/subsystem.dm [hunk 1]
```diff
@@ -255,14 +255,18 @@
 /datum/controller/subsystem/proc/OnConfigLoad()
 
 //used to initialize the subsystem AFTER the map has loaded
-/datum/controller/subsystem/Initialize(start_timeofday)
+/datum/controller/subsystem/Initialize()
 	initialized = TRUE
-	SEND_SIGNAL(src, COMSIG_SUBSYSTEM_POST_INITIALIZE, start_timeofday)
-	var/time = (REALTIMEOFDAY - start_timeofday) / 10
-	var/msg = "Initialized [name] subsystem within [time] second[time == 1 ? "" : "s"]!"
+	SEND_SIGNAL(src, COMSIG_SUBSYSTEM_POST_INITIALIZE)
```
Comment (Mothblocks) [inline code/controllers/subsystem.dm:260]: nothing relied on this giving start_timeofday

Comment (LemonInTheDark) [reply code/controllers/subsystem.dm:260]: edit the signal definition, you'll mislead people like this

Comment (Mothblocks) [reply code/controllers/subsystem.dm:260]: done

**Comment by Mothblocks** [comment]
image

**Pull request merged**